woocommerce/woocommerce
# Change Serbian currency symbol from дин. to рсд, solves #25441

### All Submissions:


### Changes proposed in this Pull Request: 

Change Serbian currency symbol to рсд from дин.

Closes #25441.

### How to test the changes in this Pull Request:

1. Head to WooCommerce >> Settings >> General.
2. Choose Serbian Dinar as a currency, the symbol should be рсд


### Other information:




### Changelog entry

> Tweak - Update serbian currency symbol to рсд from дин.
